RELEASE/scripts/autoscend/iotms/mr2021.ash

```diff
@@ -923,14 +923,17 @@ void auto_CMCconsult()
 		bestOption = 5;
 		consumableBought = $item[Breathitin&trade;];
 	}
-	else if(contains_text(page, "Homebodyl") && freeCrafts() < 5)
+	else if (!(auto_is_valid($familiar[cookbookbat]) && have_familiar($familiar[cookbookbat]) && knoll_available()) && contains_text(page, "Homebodyl") && freeCrafts() < 5)

```

Comment:
Would it be worth it to expand the `freeCrafts()` function? Thinking of something to tell us free crafts by type. Cooks, pastes, drinks. Dunno if there is one specific for drinks yet. What you have works, wondering about a more general approach

Reply:
I'll ponder it. I think this is fine for now though.